app/src/main/java/com/pokerio/app/utils/GameState.kt

```diff
@@ -139,6 +145,89 @@ object GameState {
         }
     }
 
+    fun exitSettings(
+        context: Context,
+        onError: () -> Unit,
+        onSuccess: () -> Unit,
+        baseUrl: String = BASE_URL
+    ) {
+        if (isInGame()) {
+            val sharedPreferences = context.getSharedPreferences(
+                context.getString(R.string.shared_preferences_file),
+                Context.MODE_PRIVATE
+            )
+
+            val smallBlind = sharedPreferences.getInt(
+                context.getString(R.string.sharedPreferences_small_blind),
+                1000
+            )
+
+            val startingFunds = sharedPreferences.getInt(
+                context.getString(R.string.sharedPreferences_starting_funds),
+                100
+            )
+
+            networkCoroutine.launch {
+                try {
+                    val playerID = FirebaseMessaging.getInstance().token.await()
+
+                    // Prepare url
+                    val urlString =
+                        "/modifyGame?creatorToken=$playerID&smallBlind=$smallBlind&startingFunds=$startingFunds"
+                    val url = URL(baseUrl + urlString)
+                    url.readText()
+                    ContextCompat.getMainExecutor(context).execute(onSuccess)
+                } catch (e: Exception) {
+                    e.printStackTrace()
+                    PokerioLogger.error(e.toString())
+                    ContextCompat.getMainExecutor(context).execute(onError)
+                }
+            }
+        }
+    }
```

Comment:
Please add a newline after this function

Reply:
Done.